organization/models.py

```diff
@@ -2,105 +2,145 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from model_utils import Choices
-from model_utils.fields import StatusField
-
 from users.models import User
 
-# Internal groups in KSG
-KSG_INTERNAL_GROUPS = Choices(
-    ("arrangement", "Arrangement"),
-    ("bar", "Bargjengen"),
-    ("bryggeriet", "Daglighallen mikrobryggeri"),
-    ("daglighallen", "Daglighallen"),
-    ("edgar", "Edgar"),
-    ("lychebar", "Lyche Bar"),
-    ("lychekjokken", "Lyche Kjøkken"),
-    ("sprit", "Spritgjengen"),
-    ("styret", "Styret"),
-    ("okonomi", "Økonomi"),
-)
-
-# Positions that can be held while in KSG
-# Note: Should only be available if you are aktiv
-KSG_POSITIONS = Choices(
-    # Arrangement
-    ("arrangementsansvarlig", "Arrangementsansvarlig"),
-    ("arrangementsbartender", "Arrangementsbartender"),
-    # Bargjengen
-    ("barsjef", "Barsjef"),
-    ("bartender", "Bartender"),
-    # Bryggeriet
-    ("bryggfunk", "Bryggfunk"),
-    ("brygger", "Brygger"),
-    # Daglighallen
-    ("daglighallenfunk", "Daglighallenfunk"),
-    ("daglighallenbartender", "Daglighallenbartender"),
-    # Edgar
-    ("kafeansvarlig", "Kaféansvarlig"),
-    ("baristaer", "Barista"),
-    # Lyche bar
-    ("hovmester", "Hovmester"),
-    ("barservitor", "Barservitør"),
-    # Lyche kjøkken
-    ("souschef", "Souschef"),
-    ("kokk", "Kokk"),
-    # Spritgjengen
-    ("spritbarsjef", "Spritbarsjef"),
-    ("spritbartender", "Spritbartender"),
-    # Styret
-    ("styret", "Styret"),
-    # Økonomi
-    ("okonomi", "Økonomi"),
-)
-
 
 class InternalGroup(models.Model):
-    """
-    An internal group within KSG, e.g. Lyche bar
-    """
-    STATUS = KSG_INTERNAL_GROUPS
-
-    name = StatusField(unique=True)
-
-    description = models.CharField(max_length=1024, blank=True, null=True)
+    class Meta:
+        default_related_name = 'groups'
+        verbose_name_plural = 'Internal groups'
 
-    members = models.ManyToManyField(
-        User,
-        blank=True,
-        related_name='internal_groups'  # The default django user model already has a `groups` related_name
-        # so we have to make a custom one
-    )
+    class Type(models.TextChoices):
+        """
+        Making use of https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types
+        Denotes the internal group type, either a interest-group, e.g. KSG-iT, Påfyll etc. or an internal
+        grouping, e.g. Lyche bar og Edgar
+        """
+        INTEREST_GROUP = "interest-group"
+        INTERNAL_GROUP = "internal-group"
+
```

Comment:
Have an image field which is rendered as a banner image over the description of the internal group if it exists?

Reply:
will instead have this as some future enhancement feature